config/Dockerfiles/package_tests/ansible/ansible_package_test.sh

```diff
@@ -17,6 +17,10 @@ if [[ $(grep "standard-test-beakerlib" ${package}/*.yml) == "" ]]; then
         exit 0
 fi
 if [ -f ${package}/tests.yml ]; then
+     if [[ $(ansible-playbook --list-tags ${package}/tests.yml) != *"atomic"* ]]; then
```

Comment:
Have you tested if left-hand side returns a string with spaces?

Reply:
I tried this with a real playbook both with and without the atomic tag and it worked for both so should be good